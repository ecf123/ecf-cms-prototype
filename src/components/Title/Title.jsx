import "./Title.scss";

const Title = () => {

    return (

        <div>
            <h1>Welcome Back</h1>
            <p>Lorem ipsum </p>
        </div>
        
        
    )
}

export default Title;