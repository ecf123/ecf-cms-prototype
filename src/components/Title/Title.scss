@use "../../styles/_variables.scss" as *;
@use "../../styles/typography.scss" as *;

.title {
    margin-bottom: 26px;

    &__heading {
        color: #000000;
        font-size: 32px;
        font-weight: 600;
        line-height: 38.73px;
    }

    &__text {
        color: $color-grey;
        font-size: 14px;
        font-weight: 600;
        line-height: 22.4px;
    }
    
}