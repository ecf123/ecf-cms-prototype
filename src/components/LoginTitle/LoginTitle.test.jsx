import { customRender } from "../../utils/testUtils";
import Title from "./LoginTitle";

it("should render the title", () => {
    const { container } = customRender(
    <Title>
        <h1>Welcome Back</h1>
    </Title>
    )
    expect (container).toMatchSnapshot();
}
)