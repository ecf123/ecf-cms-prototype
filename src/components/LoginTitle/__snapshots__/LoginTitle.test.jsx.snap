// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the title 1`] = `
<div>
  <div
    class="title"
  >
    <h1
      class="title__heading"
    >
      Welcome Back
    </h1>
  </div>
</div>
`;
